modules/sql-engine/src/integrationTest/sql/types/char/test_char_length.test

```diff
@@ -4,35 +4,15 @@
 
 # length shouldn't be negative;
 statement error
-SELECT 'c'::CHAR(-1);
-
-# length shouldn't be zero;
-statement error: Length for column 'C1' of type 'STRING' must be at least 1
-CREATE TABLE t_zero(c1 CHAR(0));
-
-statement ok
-CREATE TABLE t_default(c1 CHAR);
+CREATE TABLE t_negative(c1 CHAR(-1));
 
-# If <length> is omitted, then a <length> of 1 (one) is implicit.
-skipif ignite3
-# https://issues.apache.org/jira/browse/IGNITE-18492
+# length shouldn't be negative;
 statement error
-INSERT INTO t_default VALUES('12');
-
-statement ok
-INSERT INTO t_default VALUES('1');
-
-# possibility to insert empty string
-statement ok
-INSERT INTO t_default VALUES('');
-
-
-statement ok
-CREATE TABLE t_three (c1 CHAR(3), c2 CHARACTER(3));
+SELECT 'c'::CHAR(-1);
 
-# MUST FAIL due to length
+# length shouldn't be zero;
 skipif ignite3
-# https://issues.apache.org/jira/browse/IGNITE-18492
+# https://issues.apache.org/jira/browse/IGNITE-22120
```

Comment:
we cover the case by the ticket. It shouldn't be ignored

Reply:
Fixed.